components/bpmn/bpmn-explorer-web/src/web/controller/statsController.jag

```diff
@@ -25,12 +25,30 @@
         	}
 	} else {
 		include("/model/common.jag");
-		include("/model/statsModel.jag");
+
 		var update = request.getParameter("update");
+		var option = request.getParameter("option");
+
+		include("/model/statsModel.jag");
+
+		var select = request.getParameter("select");
 
 		if(update != null && update === "true"){
-			response.content = resultArray;
-		}else{
+			if( option === "userperformance"){
+				response.content = arrayOfTaskCountPerUser;
+			}
+			else if( option === "taskduration"){
+				response.content = resultArray;
+			}
+			else if( option === "avgprocessduration"){
+				response.content = processBarChartAvgTime;
+			}
+			else{
+
+			}
+		}
+		
```

Comment:
This code snippet has been moved to reportController.jag and the formatting issue is fixed there.
Fixed with commit 130406a